doc/man1/s_server.pod

```diff
@@ -335,6 +369,16 @@ The directory to use for client certificate verification. This directory
 must be in "hash format", see B<verify> for more information. These are
 also used when building the server certificate chain.
 
+=item B<-chainCApath dir>
+
+The directory to use for building the chain provided to client. This directory
+must be in "hash format", see B<verify> for more information.
```

Comment:
L\<verify\>?

Reply:
Thank you, everything you noticed about is fixed.